qiskit_sphinx_theme/__init__.py

```diff
@@ -16,6 +16,7 @@
 
 from qiskit_sphinx_theme import directives, previous_releases, translations
 
+
```

Comment:
When opening a pull request, it's a good practice to look in GitHub at the diff. Are you making any changes you didn't realize? This is a good example of one. There's no reason to be adding these blank lines here. I suspect it was leftover from when you were iterating on the code.

```suggestion
```